lib/chef/knife.rb

```diff
@@ -233,8 +233,6 @@ def self.load_deps
       end
     end
 
-    private
```

Comment:
I think this `private` didn't have the effect the author intended; I recall reading that you have to do `class << self` along with private to get private class methods. In any case, probably an `# @api private` would at least be good there?